SFS_monochromatic/driving_functions_mono/driving_function_mono_wfs_vss.m

```diff
@@ -1,20 +1,21 @@
-function D = driving_function_mono_wfs_vss(x0,xv,Dv,f,conf)
+function D = driving_function_mono_wfs_vss(x0,xv,srcv,Dv,f,conf)
 %DRIVING_FUNCTION_MONO_WFS_VSS returns the driving signal D for a given set of
 %virtual secondary sources and their driving signals
 %
-%   Usage: D = driving_function_mono_wfs_vss(x0,xv,Dv,f,conf)
+%   Usage: D = driving_function_mono_wfs_vss(x0,xv,srcv,Dv,f,conf)
 %
 %   Input parameters:
 %       x0          - position, direction, and weights of the real secondary
 %                     sources / m [nx7]
 %       xv          - position, direction, and weights of the virtual secondary
 %                     sources / m [mx7]
+%       srcv        - type of virtual secondary sources

```

Comment:
Could you add the allowed types here like for the explanation of `src` in other driving functions, e.g. `'fs' - focused source`

Reply:
fixed in 7c86d9f